src/extensions/default/SVGCodeHints/SVGUtils.js

```diff
@@ -0,0 +1,339 @@
+/*
```

Comment:
This utility file should be moved to core Brackets in the `src/language` folder in case anyone else wants to use it.

Does this only apply to SVG, or any XML? If XML, it should be XMLUtils.